router/src/server.rs

```diff
@@ -1238,8 +1239,11 @@ pub(crate) async fn chat_completions(
 
     let logprobs = logprobs.unwrap_or_default();
 
-    // static values that will be returned in all cases
-    let model_id = info.model_id.clone();
+    // extract model id from request if specified
+    let model_id = match model.as_deref() {
+        Some("tgi") | None => info.model_id.clone(),
+        Some(m_id) => format!("{}+{}", info.model_id, m_id),
```

Comment:
@philschmid that case is much more concise, however it leaves out the current model id which could obscure information from the caller. Since the generation is a combination of the model and the adapter it seems to reason to add both ids. 

I'm happy to change the id to just the adapter if that makes the most sense, however for the reasons above I think its best to include both

Reply:
ah thats a great reason to only use the adapter id, just updated in the latest commit